<launch>
  <rosparam command="load" file="$(find set_position)/config/pid_param.yaml" />
  <node name="position_controller" pkg="set_position" type="set_position_node" output="screen" />
</launch>
